ansible/inventories/devnet-10/group_vars/all/all.yaml

```diff
@@ -0,0 +1,211 @@
+
+network_subdomain: "{{ ethereum_network_name }}.ethpandaops.io"
+server_fqdn: "{{ inventory_hostname }}.srv.{{ network_subdomain }}"
+
+ethereum_network_id: >-
+  {{ (lookup('file', eth_testnet_config_local_dir_src + '/genesis.json') | from_json).config.chainId }}
+ethereum_network_deposit_contract: >-
+  {{ lookup('file', eth_testnet_config_local_dir_src + '/deposit_contract.txt') }}
+ethereum_network_deposit_contract_block: >-
+  {{ lookup('file', eth_testnet_config_local_dir_src + '/deposit_contract_block.txt') }}
+
+ethereum_node_rcp_hostname: "rpc.{{ server_fqdn }}"
+ethereum_node_beacon_hostname: "bn.{{ server_fqdn }}"
+
+ethstats_url: "ethstats.{{ network_subdomain }}"
+ethstats_secret: "{{ secret_ethstats }}"
+
+log_level: info
+
+### Checkpoint sync specific for Ansible & Kubernetes
+checkpoint_sync_node: checkpoint-sync.{{ network_subdomain }}
+checkpoint_sync_enabled: false
+checkpoint_sync_url: https://{{ checkpoint_sync_node }}
+checkpoint_sync_url_kube: https://{{ checkpoint_sync_node }}
+#  ██████╗░░█████╗░██╗░░░░░███████╗  ██╗░░░██╗░█████╗░██████╗░░██████╗
+#  ██╔══██╗██╔══██╗██║░░░░░██╔════╝  ██║░░░██║██╔══██╗██╔══██╗██╔════╝
+#  ██████╔╝██║░░██║██║░░░░░█████╗░░  ╚██╗░██╔╝███████║██████╔╝╚█████╗░
+#  ██╔══██╗██║░░██║██║░░░░░██╔══╝░░  ░╚████╔╝░██╔══██║██╔══██╗░╚═══██╗
+#  ██║░░██║╚█████╔╝███████╗███████╗  ░░╚██╔╝░░██║░░██║██║░░██║██████╔╝
+#  ╚═╝░░╚═╝░╚════╝░╚══════╝╚══════╝  ░░░╚═╝░░░╚═╝░░╚═╝╚═╝░░╚═╝╚═════╝░
+
+# role: ethpandaops.general.ethereum_genesis
+ethereum_genesis_generator_container_image: "ethpandaops/ethereum-genesis-generator:2.0.1"
+ethereum_genesis_generator_version: "{{ ethereum_genesis_generator_container_image.split(':')[-1] }}"
+ethereum_genesis_network_seed: "{{ ansible_inventory_sources[0] }}"
+ethereum_genesis_chain_id: 7011893060
+ethereum_genesis_generator_output_dir: "../network-configs/{{ ethereum_network_name | replace('dencun-', '') }}"
+ethereum_genesis_timestamp: 1697097540
+ethereum_genesis_timedelay: 60
+ethereum_genesis_timestamp_relative_cmd:
+  Linux: "date +%s -d '+45 minutes'"
+  Darwin: "date -v +30M +%s"
+ethereum_genesis_mnemonic: "{{ secret_genesis_mnemonic }}"
+ethereum_genesis_generator_config_files:
+  cl/config.yaml: "{{ lookup('ansible.builtin.url', 'https://raw.githubusercontent.com/ethpandaops/ethereum-genesis-generator/v{{ethereum_genesis_generator_version}}/config-example/cl/config.yaml', split_lines=false) }}" # noqa yaml[line-length]
+  cl/mnemonics.yaml: "{{ lookup('ansible.builtin.url', 'https://raw.githubusercontent.com/ethpandaops/ethereum-genesis-generator/v{{ethereum_genesis_generator_version}}/config-example/cl/mnemonics.yaml', split_lines=false) }}" # noqa yaml[line-length]
+  el/genesis-config.yaml: |
+    chain_id: ${CHAIN_ID}
+    deposit_contract_address: "${DEPOSIT_CONTRACT_ADDRESS}"
+    mnemonic: ${EL_AND_CL_MNEMONIC}
+    el_premine:
+      "m/44'/60'/0'/0/0": 10000000000ETH
```

Comment:
would reduce each account by a bit so we dont risk overflow. perhaps like 1B? 
And we can do 20 accounts instead so we def avoid the risk?

Reply:
If we catch overflow, I'll take that as a win. Will only reduce if we do catch an overflow tho :D

Reply:
but then its a ton of wasted time for no reason no?
cuz the scenario can never happen on mainnet, inflation isnt high enough there